pipenv/cli/command.py

```diff
@@ -342,7 +342,8 @@ def lock(
     "--fancy",
     is_flag=True,
     default=False,
-    help="Run in shell in fancy mode (for elegantly configured shells).",
+    help="Run in shell in fancy mode. Make sure the shell have no path manipulating
```

Comment:
```suggestion
    help="Run in shell in fancy mode. Make sure the shell have no path manipulating"
```
No line breaking in a quoted string, but you can use implicit string concatenation